src/styles/global.scss

```diff
@@ -32,6 +32,7 @@
   --mm-font-stack-monospace: 'Source Code Pro', monospace;
   --mm-layout-toc-width: 290px;
   --mm-layout-sidebar-width: 350px;
+  --mm-max-width: var(--mm-max-width);
```

Comment:
@wesrice I think this should be `--mm-max-width: 880px;`

Reply:
Oh bother.  *shakes hand at find and replace*